[2015-12-27 09:40:01] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ')'' in C:\xampp\htdocs\app\Http\routes.php:14
Stack trace:
#0 {main}  
[2015-12-27 09:40:57] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\ViewContorller does not exist' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php:264
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(264): ReflectionMethod->__construct('App\\Http\\Contro...', 'home')
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(838): Illuminate\Routing\Route->signatureParameters('Illuminate\\Data...')
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(823): Illuminate\Routing\Router->substituteImplicitBindings(Object(Illuminate\Routing\Route))
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(806): Illuminate\Routing\Router->substituteBindings(Object(Illuminate\Routing\Route))
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-27 09:40:58] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\ViewContorller does not exist' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php:264
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(264): ReflectionMethod->__construct('App\\Http\\Contro...', 'home')
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(838): Illuminate\Routing\Route->signatureParameters('Illuminate\\Data...')
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(823): Illuminate\Routing\Router->substituteImplicitBindings(Object(Illuminate\Routing\Route))
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(806): Illuminate\Routing\Router->substituteBindings(Object(Illuminate\Routing\Route))
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-27 09:41:29] local.ERROR: exception 'InvalidArgumentException' with message 'View [app] not found.' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:137
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(79): Illuminate\View\FileViewFinder->findInPaths('app', Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Factory.php(151): Illuminate\View\FileViewFinder->find('app')
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(732): Illuminate\View\Factory->make('app', Array, Array)
#3 C:\xampp\htdocs\app\Http\Controllers\ViewController.php(13): view('app')
#4 [internal function]: App\Http\Controllers\ViewController->home()
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(76): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(146): Illuminate\Routing\Controller->callAction('home', Array)
#7 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(94): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\ViewController), Object(Illuminate\Routing\Route), 'home')
#8 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\ViewController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'home')
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'home')
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#30 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#33 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#35 {main}  
[2015-12-27 10:09:28] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '.'' in C:\xampp\htdocs\app\Http\Controllers\ViewController.php:24
Stack trace:
#0 {main}  
[2015-12-28 14:17:03] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ')'' in C:\xampp\htdocs\app\Http\routes.php:34
Stack trace:
#0 {main}  
[2015-12-28 14:28:14] local.ERROR: exception 'ReflectionException' with message 'Method App\Http\Controllers\AccountController::() does not exist' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php:264
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(264): ReflectionMethod->__construct('App\\Http\\Contro...', '')
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(838): Illuminate\Routing\Route->signatureParameters('Illuminate\\Data...')
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(823): Illuminate\Routing\Router->substituteImplicitBindings(Object(Illuminate\Routing\Route))
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(806): Illuminate\Routing\Router->substituteBindings(Object(Illuminate\Routing\Route))
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-28 14:28:42] local.ERROR: exception 'Illuminate\Session\TokenMismatchException' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php:67
Stack trace:
#0 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#2 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#5 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#7 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#35 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#38 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#40 {main}  
[2015-12-28 14:30:51] local.ERROR: exception 'Illuminate\Session\TokenMismatchException' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php:67
Stack trace:
#0 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#2 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#5 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#7 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#35 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#38 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#40 {main}  
[2015-12-28 14:37:20] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'echo' (T_ECHO)' in C:\xampp\htdocs\storage\framework\views\43527357910f5d9146a1b520df891fcfe561f5e9.php:9
Stack trace:
#0 {main}  
[2015-12-28 14:41:45] local.ERROR: exception 'ErrorException' with message 'Undefined variable: request' in C:\xampp\htdocs\storage\framework\views\43527357910f5d9146a1b520df891fcfe561f5e9.php:9
Stack trace:
#0 C:\xampp\htdocs\storage\framework\views\43527357910f5d9146a1b520df891fcfe561f5e9.php(9): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 9, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(135): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(106): Illuminate\View\View->getContents()
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#6 C:\xampp\htdocs\storage\framework\views\7e17a04369f8604b83452567700f90cbc92ef75d.php(71): Illuminate\View\View->render()
#7 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): include('C:\\xampp\\htdocs...')
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(135): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(106): Illuminate\View\View->getContents()
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#12 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Http\Response.php(52): Illuminate\View\View->render()
#13 C:\xampp\htdocs\vendor\symfony\http-foundation\Response.php(197): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1030): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(95): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#16 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\ViewController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'home')
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'home')
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#38 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#41 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#42 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#43 {main}

Next exception 'ErrorException' with message 'Undefined variable: request (View: C:\xampp\htdocs\resources\views\app.blade.php)' in C:\xampp\htdocs\storage\framework\views\43527357910f5d9146a1b520df891fcfe561f5e9.php:9
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(44): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(135): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(106): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\storage\framework\views\7e17a04369f8604b83452567700f90cbc92ef75d.php(71): Illuminate\View\View->render()
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): include('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(135): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(106): Illuminate\View\View->getContents()
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Http\Response.php(52): Illuminate\View\View->render()
#12 C:\xampp\htdocs\vendor\symfony\http-foundation\Response.php(197): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1030): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(95): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\ViewController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'home')
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'home')
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#37 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#40 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#41 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#42 {main}

Next exception 'ErrorException' with message 'Undefined variable: request (View: C:\xampp\htdocs\resources\views\app.blade.php) (View: C:\xampp\htdocs\resources\views\app.blade.php)' in C:\xampp\htdocs\storage\framework\views\43527357910f5d9146a1b520df891fcfe561f5e9.php:9
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(44): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(135): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(106): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Http\Response.php(52): Illuminate\View\View->render()
#6 C:\xampp\htdocs\vendor\symfony\http-foundation\Response.php(197): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1030): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(95): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\ViewController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'home')
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'home')
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#31 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2015-12-28 14:44:59] local.ERROR: exception 'Illuminate\Session\TokenMismatchException' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php:67
Stack trace:
#0 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#2 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#5 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#7 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#35 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#38 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#40 {main}  
[2015-12-28 14:51:29] local.ERROR: exception 'ErrorException' with message 'Missing argument 1 for Illuminate\Support\Manager::createDriver(), called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php on line 87 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php:77
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php(77): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Missing argumen...', 'C:\\xampp\\htdocs...', 77, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php(87): Illuminate\Support\Manager->createDriver()
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php(63): Illuminate\Support\Manager->createDriver(NULL)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php(137): Illuminate\Support\Manager->driver()
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(234): Illuminate\Support\Manager->__call('getToken', Array)
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(234): Illuminate\Session\SessionManager->getToken()
#6 C:\xampp\htdocs\storage\framework\views\fe35eb371f0a19b4c0dd58263ad6763130f66a8c.php(63): csrf_token()
#7 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): include('C:\\xampp\\htdocs...')
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(135): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(106): Illuminate\View\View->getContents()
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#12 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Http\Response.php(52): Illuminate\View\View->render()
#13 C:\xampp\htdocs\vendor\symfony\http-foundation\Response.php(197): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1030): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(95): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#16 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\ViewController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'accounts')
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'accounts')
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#38 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#41 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#42 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#43 {main}

Next exception 'ErrorException' with message 'Missing argument 1 for Illuminate\Support\Manager::createDriver(), called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php on line 87 and defined (View: C:\xampp\htdocs\resources\views\pages\login.blade.php)' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php:77
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(44): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(135): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(106): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Http\Response.php(52): Illuminate\View\View->render()
#6 C:\xampp\htdocs\vendor\symfony\http-foundation\Response.php(197): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1030): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(95): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\ViewController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'accounts')
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'accounts')
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#31 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2015-12-28 14:51:30] local.ERROR: exception 'ErrorException' with message 'Missing argument 1 for Illuminate\Support\Manager::createDriver(), called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php on line 87 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php:77
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php(77): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Missing argumen...', 'C:\\xampp\\htdocs...', 77, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php(87): Illuminate\Support\Manager->createDriver()
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php(63): Illuminate\Support\Manager->createDriver(NULL)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php(137): Illuminate\Support\Manager->driver()
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(234): Illuminate\Support\Manager->__call('getToken', Array)
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(234): Illuminate\Session\SessionManager->getToken()
#6 C:\xampp\htdocs\storage\framework\views\fe35eb371f0a19b4c0dd58263ad6763130f66a8c.php(63): csrf_token()
#7 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): include('C:\\xampp\\htdocs...')
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(135): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(106): Illuminate\View\View->getContents()
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#12 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Http\Response.php(52): Illuminate\View\View->render()
#13 C:\xampp\htdocs\vendor\symfony\http-foundation\Response.php(197): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1030): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(95): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#16 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\ViewController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'accounts')
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'accounts')
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#38 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#41 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#42 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#43 {main}

Next exception 'ErrorException' with message 'Missing argument 1 for Illuminate\Support\Manager::createDriver(), called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php on line 87 and defined (View: C:\xampp\htdocs\resources\views\pages\login.blade.php)' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Manager.php:77
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(44): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(135): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(106): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Http\Response.php(52): Illuminate\View\View->render()
#6 C:\xampp\htdocs\vendor\symfony\http-foundation\Response.php(197): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1030): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(95): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\ViewController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'accounts')
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'accounts')
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#31 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2015-12-28 14:51:50] local.ERROR: exception 'Illuminate\Session\TokenMismatchException' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php:67
Stack trace:
#0 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#2 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#5 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#7 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#35 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#38 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#40 {main}  
[2015-12-28 14:56:09] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Input' not found' in C:\xampp\htdocs\app\Http\Controllers\AccountController.php:16
Stack trace:
#0 {main}  
[2015-12-28 14:58:18] local.ERROR: exception 'ErrorException' with message 'Missing argument 1 for App\Http\Controllers\AccountController::registerAccount()' in C:\xampp\htdocs\app\Http\Controllers\AccountController.php:13
Stack trace:
#0 C:\xampp\htdocs\app\Http\Controllers\AccountController.php(13): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Missing argumen...', 'C:\\xampp\\htdocs...', 13, Array)
#1 [internal function]: App\Http\Controllers\AccountController->registerAccount()
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(76): call_user_func_array(Array, Array)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(146): Illuminate\Routing\Controller->callAction('registerAccount', Array)
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(94): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\AccountController), Object(Illuminate\Routing\Route), 'registerAccount')
#5 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\AccountController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'registerAccount')
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'registerAccount')
#12 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#14 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(64): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#19 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#24 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#27 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#29 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#34 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#37 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#38 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#39 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#42 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#43 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#44 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#45 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#46 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#47 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#48 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#49 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#52 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#54 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#55 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#56 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#57 {main}  
[2015-12-28 19:54:26] local.ERROR: exception 'ReflectionException' with message 'Function () does not exist' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php:266
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Route.php(266): ReflectionFunction->__construct('')
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(838): Illuminate\Routing\Route->signatureParameters('Illuminate\\Data...')
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(823): Illuminate\Routing\Router->substituteImplicitBindings(Object(Illuminate\Routing\Route))
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(806): Illuminate\Routing\Router->substituteBindings(Object(Illuminate\Routing\Route))
#4 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-28 19:56:22] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(32): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(32): Illuminate\Support\Facades\Route::group(Array)
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(219): Illuminate\Routing\Router->group(Array, Object(Closure))
#7 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#8 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#9 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#10 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#13 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#17 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#18 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#22 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2015-12-28 19:56:23] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(32): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(32): Illuminate\Support\Facades\Route::group(Array)
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(219): Illuminate\Routing\Router->group(Array, Object(Closure))
#7 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#8 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#9 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#10 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#13 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#17 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#18 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#22 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2015-12-28 19:56:24] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(32): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(32): Illuminate\Support\Facades\Route::group(Array)
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(219): Illuminate\Routing\Router->group(Array, Object(Closure))
#7 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#8 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#9 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#10 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#13 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#17 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#18 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#22 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#24 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#27 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2015-12-28 19:56:32] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array)
#4 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#5 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#7 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#8 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#13 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#17 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#25 {main}  
[2015-12-28 19:56:32] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array)
#4 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#5 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#7 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#8 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#13 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#17 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#25 {main}  
[2015-12-28 19:56:33] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array)
#4 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#5 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#7 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#8 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#13 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#17 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#25 {main}  
[2015-12-28 19:56:33] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array)
#4 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#5 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#7 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#8 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#13 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#17 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#25 {main}  
[2015-12-28 19:56:34] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array)
#4 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#5 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#7 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#8 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#13 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#17 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#25 {main}  
[2015-12-28 19:56:34] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array)
#4 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#5 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#7 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#8 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#13 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#17 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#25 {main}  
[2015-12-28 19:56:34] local.ERROR: exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Routing\Router::group() must be an instance of Closure, none given, called in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 216 and defined' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php:370
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(370): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(4096, 'Argument 2 pass...', 'C:\\xampp\\htdocs...', 370, Array)
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(216): Illuminate\Routing\Router->group(Array)
#2 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#3 C:\xampp\htdocs\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array)
#4 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(41): require('C:\\xampp\\htdocs...')
#5 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(377): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#7 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#8 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#9 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 C:\xampp\htdocs\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#13 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#14 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#17 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#19 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#22 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#25 {main}  
[2015-12-28 21:05:56] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Http\Request::redirect()' in C:\xampp\htdocs\app\Http\Controllers\AccountController.php:49
Stack trace:
#0 {main}  
[2015-12-29 09:39:57] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'Route' (T_STRING)' in C:\xampp\htdocs\app\Http\routes.php:43
Stack trace:
#0 {main}  
[2015-12-29 09:46:00] local.ERROR: exception 'ReflectionException' with message 'Class Illuminate\Routing\Middleware\TrainerAuth does not exist' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php:736
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(736): ReflectionClass->__construct('Illuminate\\Rout...')
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(631): Illuminate\Container\Container->build('Illuminate\\Rout...', Array)
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(674): Illuminate\Container\Container->make('Illuminate\\Rout...', Array)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(123): Illuminate\Foundation\Application->make('Illuminate\\Rout...')
#4 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#5 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#6 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(64): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#8 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#9 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#13 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#14 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#18 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#19 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#24 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#27 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#29 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#31 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(705): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#34 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(678): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Routing\Router.php(654): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#37 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#42 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#44 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#45 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#46 C:\xampp\htdocs\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#47 {main}  
[2015-12-29 09:46:00] local.ERROR: exception 'ReflectionException' with message 'Class Illuminate\Routing\Middleware\TrainerAuth does not exist' in C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php:736
Stack trace:
#0 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(736): ReflectionClass->__construct('Illuminate\\Rout...')
#1 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Container\Container.php(631): Illuminate\Container\Container->build('Illuminate\\Rout...', Array)
#2 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(674): Illuminate\Container\Container->make('Illuminate\\Rout...', Array)
#3 C:\xampp\htdocs\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(152): Illuminate\Foundation\Application->make('Illuminate\\Rout...')
#4 C:\xampp\htdocs\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#5 {main}  
